cmd/root.go

```diff
@@ -130,7 +133,7 @@ func setupLoggers(logFmt string) {
 	if verbose {
 		log.SetLevel(log.DebugLevel)
 	}
-	log.SetOutput(stderr)
+	log.SetOutput(stdout)
```

Comment:
I think changing the output to `stdout` is outside of the scope for this issue.

Reply:
Thanks for the review @na-- , I have reverted to `stderr`.